edge-patron/src/main/resources/domain/edge-patron/features/patrons.feature

```diff
@@ -0,0 +1,74 @@
+Feature: patron tests
+
+  Background:
+    * url baseUrl
+    * callonce login testAdmin
+    * configure headers = {  'x-okapi-token': '#(okapitoken)', 'Accept': '*/*'  }
+    * def uuid = call random_uuid
+    * def firstName = call random_string
+    * def lastName = call random_string
+    * def owner = random_string
+    * def amount = call random_numbers
+    * def status = 'Available'
+    * def requestId = call random_uuid
+    * def servicePointId = call random_uuid
+
+  Scenario: Return total fees/fines regardless of fee/fine being attached to an item for a patron.
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostItem') { itemId: 388830d5-95db-4528-95b6-4ec9d37d4086, materialTypeId: 388830d5-95db-4528-95b6-4ec9d37d4087,materialTypeName: TestMaterial, itemBarcode: 11111}
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostItem')  { itemId: 388830d5-95db-4528-95b6-4ec9d37d4056, materialTypeId: 388830d5-95db-4528-95b6-4ec9d34d4086,materialTypeName: TestMaterial1, itemBarcode: 2222}
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostOwnerAndFine'){ materialTypeId: 388830d5-95db-4528-95b6-4ec9d34d4089, itemId: 388830d5-95db-4528-95b6-4ec9d37d4056 }
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostOwnerAndFine'){ materialTypeId: 188830d5-95db-4528-95b6-4ec9d34d4086, itemId: 388830d5-95db-4528-95b6-4ec9d37d4086 }
+    * def createUserResponse = call read('classpath:domain/edge-patron/features/util/initData.feature@PostPatronGroupAndUser') { userBarcode: 1111,username:  testUser1}
+    * def userId = createUserResponse.createUserRequest.id
+
+    Given path 'patron/account/' + userId
+    When method GET
+    Then status 200
+    And match response.totalCharges.amount == amount+amount
+
+  Scenario: Return loans for a patron
+    * def createUserResponse = call read('classpath:domain/edge-patron/features/util/initData.feature@PostPatronGroupAndUser') { userBarcode: 2222,username: testUser2}
+    * def userId = createUserResponse.createUserRequest.id
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostPolicies')
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostItem')  { itemId: 388830d5-95db-4528-95b6-4ec9d37d4057, materialTypeId: 388830d5-95db-4528-95b6-4ec9d34d4090, materialTypeName: TestMaterial2, itemBarcode: 3333}
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostCheckOut') { itemBarcode: 3333,userBarcode: 2222 }
+
+    Given path 'patron/account/' + userId
+    When method GET
+    Then status 200
+    And match response.totalLoans == 1
```

Comment:
Could it be worth checking that it is the expected loan, e.g. with the ID or barcode?

Reply:
@marcjohnson-kint :-
PFB api response we get from  **'patron/account/' + userId** ,

 {
  "charges": [
  ],
  "loans": [
  ],
  "totalCharges": {
    "amount": 0.0,
    "isoCurrencyCode": "USD"
  },
  "totalLoans": 1,
  "holds": [
  ],
  "totalChargesCount": 0,
  "totalHolds": 0
}
i think we have only one option which is **totalLoans**.

Reply:
Thanks, I didn't realise the only information included is the count